create_event.py

```diff
@@ -42,7 +42,7 @@ def create_events(events: list[Event]):
         }
 
         if event.already_in_calendar:
-            print(f"Event {event.description} is already in the calendar, skipping...")
+            print(f"Event {event.summary} is already in the calendar, skipping...")
```

Comment:
summary used as descriptions is nullable, but the summary is necessary. this is pretty much the only "non-refactor" change.